src/screens/TestCases/TestDraft/TestDraftDetails.js

```diff
@@ -79,106 +78,39 @@ function TestDraftDetails(props) {
   };
   const [modalIsOpen, setModalIsOpen] = useState(false);
   const [filterColumn, setFilterColumn] = useState(null);
+  const [filterColumnId, setFilterColumnId] = useState(null);
+
   const [modalPosition, setModalPosition] = useState({ top: 0, left: 0 });
   const [selectedFilters, setSelectedFilters] = useState([]);
-  const [filteredData, setFilteredData] = useState(
-    Array?.isArray(getDraftTestListData) ? getDraftTestListData : []
-  );
+
   const [searchTerm, setSearchTerm] = useState('');
-  // const closeModal = () => {
```

Comment:
remove all commented code, if not req. else give a proper comment

Reply:
resolved